packages/site/src/lib/components/table/Cell.svelte

```diff
@@ -79,6 +83,7 @@
       {canEdit}
       field={column.field}
       value={entry[column.field]}
+      display={$_(`entry.${column.field}`)}
```

Comment:
I don't put a default here. Is it fine?

Reply:
When using dynamic values, it's always good to have a fallback. In this case https://github.com/livingtongues/living-dictionaries/blob/c21e36599a2273295c32502983534d2fe067a0f3/packages/site/src/lib/components/modals/EditFieldModal.svelte#L13 has a fallback default of "Edit" though that's only when nothing is passed in. I'm not sure if that would work in this situation if "undefined" is passed in, I think it would overrule the fallback. You could set "Edit" as the default here.